ui/lib/pki/addon/routes.js

```diff
@@ -2,4 +2,44 @@ import buildRoutes from 'ember-engines/routes';
 
 export default buildRoutes(function () {
   this.route('overview');
+  this.route('configuration', function () {
+    this.route('index', { path: '/' });
```

Comment:
I know I don't explicitly need to state the index route here, however it helps me know that I'm relying on it instead of a file called configuration. For this reason, i'd like to keep them.